assemblies/dynamic-plugins/assembly-configuring-rhdh-plugins.adoc

```diff
@@ -20,9 +20,12 @@ include::../../artifacts/rhdh-plugins-reference/nexus-repository-manager/nexus-r
 include::../../artifacts/rhdh-plugins-reference/tekton/tekton-plugin-admin.adoc[leveloffset=+1]
 
 // Topology
-== Installing and configuring the Topology plugin
+== Installing the Topology plugin
```

Comment:
Avoid second level titles.

COnsider moving this content to a dedicated assembly file, with level 1 title.